app/helpers/charts_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
if this file not be used - please remove it

Reply:
fixed